carbonmark-api/src/routes/activities/get.schema.ts

```diff
@@ -0,0 +1,35 @@
+import { Type } from "@sinclair/typebox";
+import { ActivityModel } from "../../models/Activity.model";
+import { NetworkParamModel } from "../../models/NetworkParam.model";
+
+export const QueryString = Type.Object({
+  network: Type.Optional(NetworkParamModel),
+  projectId: Type.Optional(
+    Type.String({
```

Comment:
This should be an array of strings right? I can see the GQL query is looking through an array.

Reply:
You are right. But...
The current queries made by the ui are made like this: `http://localhost:3003/activities?activityType=Sell,Buy` and the handler converts the `Sell,Buy` string into an array
It is a pattern I copied from the projects route https://github.com/KlimaDAO/klimadao/blob/a50837f1649fe8e3ecc9c9f6f58821183c0e2779/carbonmark-api/src/routes/projects/get.ts#L42

If we want to have a proper Array type here. The UI queries must be changed to somehitng like `http://localhost:3003/activities?activityType=Sell&activityType=Buy` or change the querystring parser https://fastify.dev/docs/v2.15.x/Documentation/Server/#querystringparser

The impact seems a bit large for this PR. So I propose we do that in another one.